tensorrt_llm/evaluate/json_mode_eval.py

```
@@ -23,9 +23,31 @@
 
 from .. import LLM as PyTorchLLM
 from ..llmapi import RequestOutput
+from ..llmapi.reasoning_parser import resolve_guided_decoding_reasoning_parser
 from ..logger import logger
 from ..sampling_params import GuidedDecodingParams, SamplingParams
-from .interface import Evaluator
+from .interface import Evaluator, extract_final_content_from_generation
+
+JsonValue = Union[dict[str, object], list[object], str, int, float, bool, None]
```

Ruff (0.16.1)

[error] Undefined name `Union`

(F821)